src/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./extractor";
+export * from "./contracts";

```

Comment:
Are you sure, about exporting all stuff at one level?

Reply:
Good point. I will fix that 👍